src/utils/url.ts

```diff
@@ -9,6 +11,18 @@ export const venueInsideUrl = (venueId: string) => {
   return `/in/${venueId}`;
 };
 
+export const adminNGVenueUrl = (venueId?: string) => {
+  return generatePath("/admin-ng/venue/:venueId?", {
+    venueId: venueId,
+  });
+};
+
+export const adminNGSettigsUrl = (venueId?: string) => {
+  return generatePath("/admin-ng/advanced-settings/:venueId?", {
+    venueId: venueId,
+  });
+};
```

Comment:
```suggestion
export const adminNGSettigsUrl = (venueId?: string) => generatePath("/admin-ng/advanced-settings/:venueId?", {
    venueId: venueId,
  });
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/484c4afb77a3b370f73be7ab7dbd7911b30553c0